.github/ci_templates/tests.yml

```diff
@@ -1,7 +1,7 @@
 {!{ define "unit_run_args" }!}
 # <template: unit_run_args>
-args: 'go test -timeout=${{env.TEST_TIMEOUT}} -vet=off ./modules/... ./global-hooks/...'
-docker_options: '-w /deckhouse -v ~/go-pkg-cache:/go/pkg'
+args: 'make tests-modules'
+docker_options: '-w /deckhouse -v ${{github.workspace}}:/deckhouse -e "TERM=xterm-256color" -e "CGO_ENABLED=0" -v ~/go-pkg-cache:/go/pkg'
```

Comment:
Why CGO_ENABLED=0? It should build with default CGO_ENABLED=1 without gcc and jg.h since shell-operator 1.1.0